paasta_tools/utils.py

```diff
@@ -1871,16 +1915,30 @@ def get_zk_hosts(self) -> str:
             return hosts[len("zk://") :]
         return hosts
 
-    def get_system_docker_registry(self) -> str:
-        """Get the docker_registry defined in this host's cluster config file.
+    def get_push_docker_registries(self) -> List[str]:
+        """Get the 'push_registries' defined in this host's cluster config file.
+        If not present, uses 'docker_registry' as an alternative.
 
-        :returns: The docker_registry specified in the paasta configuration
+        :returns: The 'push_registries' specified in the paasta configuration or
+        the array consisting of 'docker_registry'.
         """
+        try:
+            return self.config_dict["push_registries"]
+        except KeyError:
```

Comment:
rather than fallback to the `docker_registry` key here, can you put this in the caller please

Reply:
Oh, I see. Thanks, fixed 👍